### ISSUE TITLE
[next][pyodide] Use the new pyodide feature for better tracebacks

### ISSUE THREAD

**Issue by WebReflection** [pyscript/pyscript#1701]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

Coming from this issue https://github.com/pyscript/polyscript/issues/36 ... once *Pyodide* is out we should consider using this feature to have better errors / stack traces.

**Comment by WebReflection** [comment]
This is blocked by https://github.com/pyodide/pyodide/issues/4153#issuecomment-1725125095

**Comment by WebReflection** [comment]
I did investigate this a little bit and it's clear to me this is indeed something *polyscript* related so I am closing this in favor of this one https://github.com/pyscript/polyscript/issues/36
